src/renderer/components/opds/Header.tsx

```diff
@@ -23,17 +23,21 @@ export enum DisplayType {
     List = "list",
 }
 
-interface HeaderProps extends RouteComponentProps, TranslatorProps {
-    displayType?: DisplayType;
+interface IProps extends RouteComponentProps, TranslatorProps {
+    displayType: DisplayType;
 }
 
-export class Header extends React.Component<HeaderProps, undefined> {
+class Header extends React.Component<IProps> {
     public render(): React.ReactElement<{}> {
         const { __ } = this.props;
         const search = parseQueryString(this.props.location.search.replace("?", ""));
         const displayType = search.displayType || DisplayType.Grid;
         delete(search.displayType);
 
+        /**
+         * Why css style is apply in code and not imported from css ressource ?
+         * code:cssincode
+         */
```

Comment:
Is the `code:cssincode` marker a known syntax? (i.e. recognized by Visual Studio Code, or a plugin?)
If not, may I suggest using `TODO` or `FIXME`, with a comment like `extract inline CSS into https://github.com/readium/readium-desktop/blob/develop/src/renderer/assets/styles/opds.css` (for example)

Reply:
it's comment code to easily find all css in code with simple `ctrl` `f`.
I added a `noany` also.

Is it no conventional ?
I can replace it with a simple TODO instead

Reply:
`TODO` and `FIXME` are commonly-used comment tokens to mark code locations that need addressing.
https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight
https://marketplace.visualstudio.com/items?itemName=TzachOvadia.todo-list
https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree
https://www.jetbrains.com/help/idea/using-todo.html
https://docs.microsoft.com/en-us/visualstudio/ide/using-the-task-list?view=vs-2019
etc.

Reply:
thank you daniel

Reply:
Sorry, I do not understand your last message. Does this mean you want to replace the `code:cssincode` (etc.) notation with `TODO` / `FIXME` in *this PR*, or that you wish to implement the correction in *another PR*?

Reply:
https://github.com/readium/readium-desktop/pull/696/commits/25ec2d25ee457dcf766a51a0cd0e42a5be7ca17b